M2/Macaulay2/d/evaluate.d

```diff
@@ -1194,15 +1194,20 @@ globalAssignmentFun(x:globalAssignmentCode):Expr := (
      globalAssignment(t.frameindex,t,newvalue));
 
 ParallelAssignmentError(n:int):Expr := buildErrorPacket(
-    "parallel assignment: expected a sequence of " + tostring(n) + " values");
+    "parallel assignment: expected a sequence of " + tostring(n) + " value(s)");
```

Comment:
Not worth a separate commit, but in the next push this could be changed to `parallel assignment expected a sequence of length N` (note I removed the colon also).

Reply:
Or perhaps `parallel assignment to N symbols encountered a sequence of length M`